tests_new/integration_tests/modules/vote/test_vote.py

```diff
@@ -39,13 +39,5 @@ def test_count_upvote_invalid(client1, vote1, session_s3_dataset1):
     )
 
 
-def test_count_votes(client2, vote1, session_s3_dataset1):
-    count = count_upvotes(client2, session_s3_dataset1.datasetUri, S3_DATASET_TARGET_TYPE)
-
-    # Assert incremeent by 1
-    upvote(client2, session_s3_dataset1.datasetUri, S3_DATASET_TARGET_TYPE, True)
-    assert_that(count_upvotes(client2, session_s3_dataset1.datasetUri, S3_DATASET_TARGET_TYPE)).is_equal_to(count + 1)
-
-    # Assert decrement by 1
-    upvote(client2, session_s3_dataset1.datasetUri, S3_DATASET_TARGET_TYPE, False)
-    assert_that(count_upvotes(client2, session_s3_dataset1.datasetUri, S3_DATASET_TARGET_TYPE)).is_equal_to(count)
+def test_count_votes(client1, vote1, session_s3_dataset1):
+    assert_that(count_upvotes(client1, session_s3_dataset1.datasetUri, S3_DATASET_TARGET_TYPE)).is_equal_to(1)
```

Comment:
Don't we also need a test, that for client2 in fails due to lack of permissions?

Reply:
I think we cover a lot of those unauth tests in the functional tests

Reply:
No because count is public, there are no restrictions on who can count the votes